scripts/cluster/yarn_cluster_setup/move_files_to_master.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
This is a script for moving things in the cluster nodes. We should put it outside this folder. Also, in addition to the master node, we should copy files to all nodes. This can be achieved by git cloning on these node the SparkFHE-Addon repo.

Reply:
After consulting with @bharathbrat I have removed the move script altogether. It is assumed that all the relevant folders will be available in spark-3.0.0-SNAPSHOT-bin-SparkFHE.